### PR TITLE
Fixing failing tests.

### LINKED ISSUE #208: Tests failing with SBCL 2.4.5
Hey, I'm packaging CLX for Void Linux and I got 2 failing tests:

```
 Test run had 3 failures:

  Failure 1: FAILED-ASSERTION when running XLIB-TEST-DISPLAYS::GRAPHICS-CONTEXTS
    FINISHES block did not finish: FIASCO:FINISHES

  Failure 2: UNEXPECTED-ERROR when running XLIB-TEST-DISPLAYS::GRAPHICS-CONTEXTS
    The value
      :ON-OFF-DASH
    is not of type
      (MEMBER :DOUBLE-DASH :DASH :SOLID)
    when binding XLIB::VALUE

  Failure 3: FAILED-ASSERTION when running XLIB-TEST-DISPLAYS::WINDOWS-AND-PIXMAPS
    Binary predicate (= X Y) failed.
    x: (XLIB:DRAWABLE-WIDTH XLIB-TEST-DISPLAYS::CHILD-WINDOW) => 120
    y: 100 => 100
```

can somebody help? Full build log is here https://github.com/void-linux/void-packages/actions/runs/9357745671/job/25758175185?pr=49434

### PR DESCRIPTION
**Pull request by MIvanchev**
Fixes #208.

### CODE REVIEW

File: gcontext.lisp
Comment (metayan) [inline gcontext.lisp:1]: Good.
should also add the change in the manual:
```diff
diff --git a/manual/clx.texinfo b/manual/clx.texinfo
index 7a90a09..354553e 100644
--- a/manual/clx.texinfo
+++ b/manual/clx.texinfo
@@ -5804,7 +5804,7 @@ ends of the individual dashes, except @var{:not-last} is treated as
 
 @table @var
 @item line-style
-One of @var{:dash}, @var{:double-dash}, or @var{:solid}.
+One of @var{:on-off-dash}, @var{:double-dash}, or @var{:solid}.
 @end table
 
 @end defun
```

File: tests/core-protocol.lisp
Comment (metayan) [inline tests/core-protocol.lisp:1]: Works.
I also removed the `(sleep 2)` and it worked fine. No idea why it's there.

File: gcontext.lisp
Comment (MIvanchev) [reply gcontext.lisp:1]: Done! Thanks.

File: tests/core-protocol.lisp
Comment (MIvanchev) [reply tests/core-protocol.lisp:1]: Removed.

**Comment by dkochmanski** [comment]
thank you both for the pull request and the review; lgtm

**Comment by JMC-design** [comment]
Why not keep dash, as on-off-dash is just a long winded way of writing the definition of dash before the word dash?

**Comment by dkochmanski** [comment]
Indeed, I've changed it back to `:dash` and updated tests. Most notably this change is not backward compatible.

**Pull request merged**